sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -1138,20 +1138,36 @@ def clear_state(self, window, tag):
     if not self.state[window]:
       self.state.pop(window, None)
 
-  def get_timers(self, clear=False, watermark=MAX_TIMESTAMP):
+  def get_timers(self, clear=False, watermark=MAX_TIMESTAMP, current_time=None):
+    """Gets expired timers and reports if there
```

Comment:
<!--new_thread; commit:92ae785f427ba708614ab0ad110b465142150610; resolved:0-->
Let's try to fit in one line:

```python
"""Returns expired timers and existence of REAL_TIME timers."""
```